third_party/amd/lib/TritonAMDGPUToLLVM/BufferOpsEmitter.cpp

```diff
@@ -33,6 +33,72 @@ bool isZero(Value v) {
   }
   return false;
 }
+
+// Create the auxiliary/cachepolicy value of ROCDL::RawPtrBufferLoad/StoreOp
```

Comment:
This is definitely looking nice! My feeling is that I would still like a bit more explanation:
- What does it mean a `{group,device,system,wave}` cache level? Are we enabling those systems? And why there are `x` in some cases?
- Maybe in `getCacheModifierForTarget` you can put a description of what `.ca`, `.cg`, `.cs`, etc... mean?

This is not a hard requirement for though, and I would also like other people to comment (cc @guacamoleo @sjw36 )

Reply:
I'm not entirely sure how the cache levels (wave, group, device, system) factor into this, but the different combinations of the 3 bits seem to best align with the behavior of NVIDIA cache modifiers. Some bits are 'x' because their values do not affect the cache behavior.